css/index.css

```diff
@@ -0,0 +1,359 @@
+/* --- Local styles --- */
```

Comment:
[OPTIONAL] The first few books that someone adds are getting a little clamp on top of each other, maybe you would like to look into that and improve upon the current styles.

![image](https://user-images.githubusercontent.com/79114233/148955886-77757f1d-87f0-4361-b0b7-903f868fcd41.png)